apps/src/templates/sectionProgress/sectionProgressRedux.js

```diff
@@ -57,6 +61,23 @@ export const jumpToLessonDetails = (lessonOfInterest) => {
     dispatch(setCurrentView(ViewType.DETAIL));
   };
 };
+export const processScriptAndProgress = (scriptId) => {
+  return (dispatch, getState) => {
+    const state = getState().sectionProgress;
+    const studentLevelProgress = state.studentLevelProgressByScript[scriptId];
+    const scriptData = state.scriptDataByScript[scriptId];
+    let levelsByStudentByLesson = {};
```

Comment:
Isn't this `levelsByLessonByStudent`? The top-level keys are student ids, and the values are the results of `levelsByLesson`.